langworld_db_data/movers/listed_value_mover.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+from langworld_db_data.adders.listed_value_adder import ListedValueAdder
+from langworld_db_data.constants.literals import ID_SEPARATOR
+from langworld_db_data.filetools.csv_xls import read_dicts_from_csv, write_csv
+from langworld_db_data.movers.mover import Mover, MoverError
+from langworld_db_data.removers.listed_value_remover import ListedValueRemover
+
+
+class ListedValueMoverError(MoverError):
+    pass
+
+
+class ListedValueMover(Mover):
+    def __init__(  # type: ignore
+        self,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.listed_value_adder = ListedValueAdder(
+            input_file_with_listed_values=self.output_file_with_listed_values,  # Otherwise in tests
+            # listed_value_adder opens the input inventory where the removed value is
+            # present and throws an error
+            input_dir_with_feature_profiles=self.output_dir_with_feature_profiles,
+            output_file_with_listed_values=self.output_file_with_listed_values,
+            output_dir_with_feature_profiles=self.output_dir_with_feature_profiles,
+        )
+        self.listed_value_remover = ListedValueRemover(
+            **kwargs,
+        )
+
+    def move_listed_value(
+        self,
+        initial_value_id: str,
+        index_to_assign: int,
+    ):
+        if int(initial_value_id.split("-")[2]) == index_to_assign:
+            raise ListedValueMoverError("Initial and final indices must not coincide.")
+        value_to_move = self.listed_value_remover.remove_listed_value(initial_value_id)
+        print(index_to_assign)
```

Comment:
```suggestion
```
I guess that should be removed?

Reply:
Done! I also checked for other debugging prints, none were found